lattices.nix

```diff
@@ -0,0 +1,27 @@
+# this configuration is contrib, if it doesn't work anymore, please fix it and submit a PR
+{ mkDerivation, base, base-compat, containers, deepseq, hashable
+, integer-logarithms, QuickCheck, quickcheck-instances
+, semigroupoids, stdenv, tagged, tasty, tasty-quickcheck
+, transformers, universe-base, universe-reverse-instances
+, unordered-containers
+}:
+mkDerivation {
+  pname = "lattices";
+  version = "2.0.2";
```

Comment:
will this break when I bump a version?

I'm curious how fast someone would fix it.

Reply:
I generated this file with `cabal2nix lattices.cabal > lattices.nix`. It's easy to re-generate if needed. The version there is actually irrelevant. It's only metainformation for nixpkgs and this file won't be used by nixpkgs anyway. The only time a user would have to regenerate it is when dependencies in `lattices.cabal` change.

Reply:
I have more questions, but maybe it's better to not ask them.